src/lib/eliom_registration.client.ml

```diff
@@ -204,8 +204,9 @@ module Html = Make (struct
     let reset_reload_fun = false
 
     let send ?options:_ page =
-      Eliom_client.set_content_local
-        (Eliom_content.Html.To_dom.of_element page)
+      Eliom_client_core.with_new_page ~replace:false () @@ fun () ->
```

Comment:
Do we need both this and the call to `Eliom_client_core.with_new_page` in `Eliom_client.change_page`?